src/main/kotlin/blackjack/controller/BlackjackController.kt

```diff
@@ -0,0 +1,58 @@
+package blackjack.controller
+
+import blackjack.domain.BlackjackGame
+import blackjack.domain.Challenger
+import blackjack.domain.Challengers
+import blackjack.view.InputView
+import blackjack.view.ResultView
+
+class BlackjackController(
+    private val inputView: InputView = InputView(),
+    private val resultView: ResultView = ResultView(),
+) {
+    fun play() {
+        val challengers = inputView.inputPlayerNames()
+            .map { Challenger(it) }
+            .let { Challengers(it) }
```

Comment:
이건 사람마다 취향이 다를 것 같은데 조심스럽게 의견 남겨보면 nullable한 값이라면 let으로 체크를 한번 수행한 뒤 변환하는게 자연스러울 것 같지만 여기선 아래와 같은 형태가 조금 더 가독성이 좋지 않을까 하는 의견이 있네요.

```suggestion
        val challengers = Challengers(inputView.inputPlayerNames().map { Challenger(it) })
```

한줄이 조금 길어지긴 하지만 map으로 `List<Challenger>`로 변경되는것이 명확하게 드러나며 일급 컬렉션의 생성자가 외부에 감싸져 있어 저는 좀 더 가독성이 좋게 느껴지는 것 같습니다.

중요한 부분은 아니니 그냥 참고만 부탁드려요~